charts/spire/charts/spire-server/templates/statefulset.yaml

```diff
@@ -137,6 +137,45 @@ spec:
               mountPath: /tmp
               readOnly: false
         {{- end }}
+
+        {{- if eq (.Values.tornjak.enabled | toString) "true" }}
+        - name: tornjak
+          # image: {{ .Values.tornjak.config.image }}:{{ .Values.tornjak.config.version }}
+          image: {{ template "spire-server.image" (dict "appVersion" $.Chart.AppVersion "image" .Values.tornjak.image) }}
+          imagePullPolicy: {{ .Values.tornjak.image.pullPolicy }}
+          startupProbe:
+            httpGet:
+              scheme: HTTP
+              port: 3000  
+            failureThreshold: 6
+            initialDelaySeconds: 60
+            periodSeconds: 30
+            successThreshold: 1
+            timeoutSeconds: 10
+          env:
+          {{- if .Values.tornjak.config.frontend }}
```

Comment:
If the frontend is just a webserver that calls the backend, is there any reason it shouldn't be its own deployment or maybe even chart?